controllers/followershipController.js

```diff
@@ -0,0 +1,195 @@
+import models from '../database/models';
+
+const { User, Follower } = models;
+
+/**
+ * @exports
+ * @class Followership
+ *
+ */
+export default class Followership {
+  /**
+     * @static
+     *
+     * @description allows a user follow another user
+     *
+     * @param {object} req request object
+     * @param {object} res response object
+     * @returns {object} returns follow data
+     *
+     * @memberof Followership
+     */
+  static async follow(req, res) {
+    const { id } = req.params;
+    const { id: userid } = req.user;
+
+    const foundUser = await User.findOne({ where: { id } });
+    const foundMe = await User.findOne({ where: { id: userid } });
+    if (!foundUser) {
+      return res.status(404).send({
+        errors: {
+          message: 'This author does not exist',
+        }
+      });
+    }
+    const { isVerified } = foundMe;
+    if (!isVerified) {
+      return res.status(400).send({
+        errors: {
+          message: 'You are supposed to verify your account to follow your favourite authors',
+        }
+      });
+    }
+    if (id === userid) {
+      return res.status(400).send({
+        errors: {
+          message: 'You are not allowed to follow yourself',
+        }
+      });
+    }
+
+    await Follower.findOrCreate({ where: { followerId: userid, userId: id } })
+      // eslint-disable-next-line no-unused-vars
```

Comment:
Why is this here?

Reply:
there's a follow argument that i'm not using in the then block that returns the followers id and users id, and i left it there incase someone may need it in future

Reply:
You can take it out, the future will take care of itself

Reply:
Have you taken out this "unused-var"?

Reply:
yes i have